src/app/components/Clipboard/ClipboardButton.test.tsx

```diff
@@ -24,11 +24,9 @@ describe("ClipboardButton", () => {
 
 		expect(screen.queryByTestId("clipboard-button__checkmark")).not.toBeInTheDocument();
 
-		await act(async () => {
-			fireEvent.click(screen.getByTestId("clipboard-button__wrapper"));
-		});
+		fireEvent.click(screen.getByTestId("clipboard-button__wrapper"));
 
-		expect(screen.queryByTestId("clipboard-button__checkmark")).toBeInTheDocument();
+		expect(await screen.findByTestId("clipboard-button__checkmark")).toBeInTheDocument();
```

Comment:
```suggestion
		await expect(screen.findByTestId("clipboard-button__checkmark")).resolves.toBeInTheDocument();
```
Same for all of the others or it'll result in brittle expectations.

Reply:
Fixed in https://github.com/PayvoHQ/wallet/pull/498/commits/92099cceb6d753e8ea4e1fa7d16756fb19fcdd75